src/hb-ot-os2-table.hh

```diff
@@ -218,6 +209,15 @@ struct OS2
   font_page_t get_font_page () const
   { return (font_page_t) (version == 0 ? fsSelection & 0xFF00 : 0); }
 
+  unsigned get_size () const
+  {
+    unsigned result = min_size;
+    if (unlikely (version >= 1)) result += v1X.get_size ();
```

Comment:
These unlikely's are not justified.  Unlikely should be used for exceptional paths.

Reply:
Fixed.